### DISCUSSION TITLE
Requesting/Adding dialogue options?

### DISCUSSION THREAD

**Discussion by WiiBii2** [Monika-After-Story/MonikaModDev#7846] [QA - code and content creation (technical)] [answered]
Hello! I have been playing Monika After Story and I was wondering what the process was for writing and submitting random dialogue options for Monika? I feel that people tend to run through the ones that are already in game too fast, and would like to help by adding my own! Thanks in advance!

**Comment by multimokia** [comment, marked as answer]
Hi there, thanks for your interest in contributing to MAS!

You'll want to have a visit to our [Contributing Guidelines](https://github.com/Monika-After-Story/MonikaModDev/wiki/Contributing-Guidelines) to get a rundown on creating and testing additions to MAS.

As well, a useful resource to get you familiar with a bit of the structure would be our Wiki's [Dialogue Coding](https://github.com/Monika-After-Story/MonikaModDev/wiki/Dialogue-Coding) section.
